### PR TITLE
asm.js simd.js : Remove #ifndef INTERPRETER_ASMJS to support ext ops in asmjs

### LINKED ISSUE #888: Asm.js Simd.js : Handling of extended opcodes with Medium or large layouts is disabled in InterpreterLoop.inl for Asm.js
Handling of extended opcodes with Medium or large layouts is disabled in InterpreterLoop.inl. This will cause a crash when interpreter SIMD opcodes (extended ones) with functions with over 256 bytecode registers.

https://github.com/Microsoft/ChakraCore/pull/831/files/5dfff42fdd6fe01a8b5680dc0b1e89b827594370#r61509222 (see for more information)

### PR DESCRIPTION
**Pull request by Krovatkin**
Fixes #888

### CODE REVIEW

**Comment by nmostafa** [comment]
Fixes #888

**Comment by Krovatkin** [comment]
@nmostafa added an unittest exposing the original issue.

**Comment by dilijev** [comment]
@Krovatkin looks like you're missing a copyright header in your test file:

`test/Bugs/testSimdManyVars.js ... does not contain a correct Microsoft copyright notice.`

File: test/Bugs/testSimdManyVars.js [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+    var f4fromUint32x4 = f4.fromUint32x4;
+    var f4fromInt32x4Bits = f4.fromInt32x4Bits;
+    
+    var f4fromInt16x8Bits = f4.fromInt16x8Bits;
+    var f4fromInt8x16Bits = f4.fromInt8x16Bits;
+    var f4fromUint32x4Bits = f4.fromUint32x4Bits;
+    var f4fromUint16x8Bits = f4.fromUint16x8Bits;
+    var f4fromUint8x16Bits = f4.fromUint8x16Bits;
+
+
```
Comment (dilijev) [inline test/Bugs/testSimdManyVars.js]: nit: Please remove duplicate blank lines and trailing whitespace throughout this test.

Comment (Krovatkin) [inline test/Bugs/testSimdManyVars.js]: @dilijev parachutes :-) lemme add the copyright header and deal with whitespaces

**Comment by Krovatkin** [comment]
@dilijev done

File: test/Bugs/testSimdManyVars.js [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+    var i4equal = i4.equal;
+    var i4greaterThan = i4.greaterThan;
+    var i4select = i4.select;
+    var i4and = i4.and;
+    var i4or = i4.or;
+    var i4xor = i4.xor;
+    var i4not = i4.not;
+    //var i4shiftLeftByScalar = i4.shiftLeftByScalar;
+    //var i4shiftRightByScalar = i4.shiftRightByScalar;
+    //var i4shiftRightArithmeticByScalar = i4.shiftRightArithmeticByScalar;
```
Comment (dilijev) [inline test/Bugs/testSimdManyVars.js]: Why are these commented-out lines in this test?

File: test/Bugs/testSimdManyVars.js [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+
+    var u16 = stdlib.SIMD.Uint8x16;
+    var u16check = u16.check;
+
+    var i16 = stdlib.SIMD.Int8x16;
+    var i16check = i16.check;
+
+    var fround = stdlib.Math.fround;
+
+
```
Comment (dilijev) [inline test/Bugs/testSimdManyVars.js]: nit: remove extra blank line here

**Comment by dilijev** [comment]
Otherwise LGTM

**Comment by Krovatkin** [comment]
@dilijev fixed formatting errors. Do I need to squash my commits into one?

**Comment by dilijev** [comment]
@Krovatkin Generally we prefer that, yes. You can have more than one commit if there is a logical division for someone reading the history.

**Comment by Krovatkin** [comment]
@dilijev done! Unfortunately, this PR won't qualify for a free-fries promo anymore :-)

**Comment by dilijev** [comment]
LGTM, although I haven't done a review of the actual content.

**Comment by nmostafa** [comment]
@MikeHolman , can you take a look please ?

**Comment by MikeHolman** [comment]
lgtm

**Pull request merged**